src/main/java/com/mabl/integration/jenkins/domain/ApiResult.java

```diff
@@ -0,0 +1,4 @@
+package com.mabl.integration.jenkins.domain;
+
+public abstract class ApiResult {
+}
```

Comment:
Why an empty abstract class than an interface?

Reply:
yeah this should be an interface. I originally thought all those objects had more in common because they are all api result set mappings, but that's not the important defining relationship.

It's that they're all unrelated types of mabl objects that should have the ApiResult behavior supplied.